driver-core/src/main/java/com/datastax/driver/core/Requests.java

```diff
@@ -31,6 +31,9 @@ private Requests() {}
   static class Startup extends Message.Request {
     private static final String CQL_VERSION_OPTION = "CQL_VERSION";
     private static final String CQL_VERSION = "3.0.0";
+    private static final String DRIVER_VERSION_OPTION = "DRIVER_VERSION";
+    private static final String DRIVER_NAME_OPTION = "DRIVER_NAME";
+    private static final String DRIVER_NAME = "DataStax Java Driver for Apache Cassandra";
```

Comment:
Should we add `Apache Cassandra (R)` or am I being paranoid?

Reply:
Follow up: We just chatted about this and decided to just use "DataStax Java Driver" for now.  Including "Apache Cassandra" is probably over descriptive, since this information will be surfaced in nodetool and virtual table output, so it doesn't really add anything.